src/utils/date.ts

```diff
@@ -356,6 +358,8 @@ const languageToLocale = (language: Language): Locale => {
       return nb;
     case Language.English:
       return en;
+    case Language.Nynorsk:
+      return nb;
```

Comment:
Humanize Duration that we use [does not seem to support nynorsk, but it can be added](https://www.npmjs.com/package/humanize-duration#supported-languages). Creating new issue for this, aiming to solve before FRAM release.